quiche/src/lib.rs

```diff
@@ -7013,6 +7017,7 @@ impl<F: BufFactory> Connection<F> {
             reset_stream_count_remote: self.reset_stream_remote_count,
```

Comment:
Odd that Stats does not contain RTT information or other information stored in the recovery object.

Reply:
Some rtt stats seem to be exposed [here](https://github.com/cloudflare/quiche/blob/master/tokio-quiche/src/quic/connection/mod.rs#L84) as part of [PathStats](https://github.com/cloudflare/quiche/blob/master/quiche/src/path.rs#L883-L894)